server/webhook.go

```diff
@@ -818,12 +818,12 @@ func (p *Plugin) handleCommentAuthorNotification(event *github.IssueCommentEvent
 	message := ""
 	switch splitURL[len(splitURL)-2] {
 	case "pull":
-		message = "[%s](%s) commented on your pull request [%s#%v](%s)"
+		message = "[%s](%s) %s comment on your pull request [%s#%v](%s)"
```

Comment:
Maybe 
```suggestion
		message = "[%s](%s) %s a comment on your pull request [%s#%v](%s)"
```

Then the message would be e.g.
`someone deleted a comment on your pull request`

Reply:
Let's discuss first, if we want this.